tests/utils/zephyr/test_plane_shift.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2025 D-Wave
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# ================================================================================================
+
+
+import unittest
+from itertools import combinations
+
+from parameterized import parameterized
+
+from minorminer.utils.zephyr.plane_shift import PlaneShift, ZPlaneShift
+
+
+class TestPlaneShift(unittest.TestCase):
+    def setUp(self) -> None:
+        self.shifts = [
+            (0, 1),
+            (-1, 0),
+            (2, -1),
+            (-4, 6),
+            (10, 4),
+            (0, 0),
+        ]
+
+    def test_valid_input_runs(self) -> None:
+        for shift in self.shifts:
+            PlaneShift(*shift)
```

Comment:
Looks like you're testing construction here:

```suggestion
    def test_construction(self) -> None:
        for shift in self.shifts:
            p = PlaneShift(*shift)
            self.assertEqual(shift, (p.x, p.y))
```